packages/fe-fpm-writer/test/unit/__snapshots__/column.test.ts.snap

```diff
@@ -291,20 +291,6 @@ exports[`CustomAction generateCustomColumn version 1.85, no handler, all propert
 
 exports[`CustomAction generateCustomColumn version 1.85, no handler, no fs, all properties 1`] = `
 Object {
-  "@com.sap.vocabularies.UI.v1.LineItem": Object {
-    "columns": Object {
-      "NewCustomColumn": Object {
-        "availability": "Adaptation",
-        "header": "col header",
-        "position": Object {
-          "anchor": "DataField::BooleanProperty",
-          "placement": "After",
-        },
-        "template": "sapux.fe.fpm.writer.test.extensions.custom.NewCustomColumn",
-        "width": "150px",
-      },
-    },
-  },
```

Comment:
@stefanschreck 

test `version 1.85, no handler, no fs, all properties`.
In test we are adding new custom column `NewCustomColumn`:
![image](https://user-images.githubusercontent.com/90789422/198263310-c0ea6c52-63cd-4d70-b7b2-232afae33695.png)

but in test expectation `NewCustomColumn` is not appearing. I think `NewCustomColumn` should appear in test snapshot for `version 1.85, no handler, no fs, all properties` like it was before

Reply:
Thanks, Andis. Yes, one of the tests needed a correction.